engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/Module.scala

```diff
@@ -0,0 +1,129 @@
+package org.enso.compiler.core.ir
+
+import org.enso.compiler.core.IR
+import org.enso.compiler.core.IR.{Identifier, randomId, ToStringHelper}
+import org.enso.compiler.core.ir.module.scope.{Definition, Export, Import}
+
+/** A representation of a top-level Enso module.
+  *
+  * Modules may only contain imports and top-level bindings, with no top-level
+  * executable code.
+  *
+  * @param imports     the import statements that bring other modules into scope
+  * @param exports     the export statements for this module
+  * @param bindings    the top-level bindings for this module
+  * @param location    the source location that the node corresponds to
+  * @param passData    the pass metadata associated with this node
+  * @param diagnostics compiler diagnostics for this node
+  */
+@SerialVersionUID(
+  6655L // SuggestionBuilder needs to send ascribedType of constructor parameters
```

Comment:
This value shall probably change to `7681`, but I am not afraid that the old caches would deserialize after all the class renames...